internal/cmd/request.go

```diff
@@ -115,3 +123,71 @@ func PrintCMDResponseFull(request any, response core.APIResponse, responseType a
 		fmt.Println(util.MustToJSONIndent(responseContent))
 	}
 }
+
+// Attempt to convert an API response to a TSV table.
+// Returns false if there are issues or the TSV table if it was successful.
```

Comment:
This statement is not correct.
You are returning two things, so this line cannot be correct.